src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -31,10 +31,12 @@ export interface ConnectorTransaction {
   /**
    * Settle date in UNIX milliseconds
    */
-  settleDate: number;
+  settleDate: number | null;
+  creationDate?: number;
```

Comment:
which connectors do not have a creation date? can't this be taken from the bolt11 invoice?

Reply:
all other connectors only pass settled transactions and we don't even need creationDate for them except for nwc. so i kept it optional and passed this field only for nwc connector. maybe we can be consistent accross and pass it for all now

Reply:
for commando there is no creation date passed via api. https://docs.corelightning.org/reference/lightning-listinvoices

Reply:
it can be taken from the bolt11 invoice, can't it?

Reply:
resolved. added creationDate for all connectors.